src/zenml/integrations/label_studio/label_studio_utils.py

```diff
@@ -18,6 +18,13 @@
 from urllib.parse import quote, urlparse
 
 
+def clean_url(url: str) -> str:
+    """Remove extraneous parts of the URL prior to mapping."""
```

Comment:
I don't really get what this does from neither code nor docstring. Maybe a simple example in the docstring could be helpful